handler/status.go

```diff
@@ -65,6 +67,14 @@ func Status(
 				"value": func(f float64) string {
 					return strconv.FormatFloat(f, 'f', -1, 64)
 				},
+				"join": func(base string, paths ...string) string {
```

Comment:
Instead of passing in a whole new function, wouldn't it be easier to just pass in the pre-calculated complete string to prefix URLs in the template with? The function is always called in the same way in the template.

Reply:
I can do that. The general purpose function is useful for future uses, but those uses don't currently exist, and are unlikely to be drastically different I'm guessing.